actionview/lib/action_view/helpers/sanitize_helper.rb

```diff
@@ -129,7 +129,7 @@ def sanitizer_vendor
         end
 
         def sanitized_allowed_tags
-          safe_list_sanitizer.allowed_tags
+          safe_list_sanitizer.class.allowed_tags
```

Comment:
Can you make similar change to `sanitized_allowed_attributes` as well? It would also fail when `ActionView::Base.sanitized_allowed_attributes` is used.

Reply:
@abhaynikam will do!

Reply:
It looks like this was broken by https://github.com/rails/rails/commit/52f0b050e25cac6d9571d71c9f74ea583d8aa2b0#diff-4c1f3e49dbd40e9f860377cac33e1a0eR128-R134.  Since `sanitizer_vendor` seems to be here to stay, I'd be in favor or restoring the original call style:

```suggestion
          sanitizer_vendor.safe_list_sanitizer.allowed_tags
```

Also, perhaps we should add some tests to ensure these methods don't break again in the future?  We don't have to assert on their return values -- just a simple `assert_nothing_raised`.

Reply:
@jonathanhefner good points! Thanks, will do

Reply:
@jonathanhefner done! hopefully this is a proper way to test class method mixins? Sorry, not so familiar with minitest